go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/kubermatic/utils
 
 go 1.14
 
+replace k8s.io/client-go => k8s.io/client-go v0.18.5
```

Comment:
why is this needed? And it worked previously.

Reply:
I think kustomize somehow depends on client-go? because now code cannot be compiled due to some issues of `k8s.io/client-go v11.0.0+incompatible`, and in KubeCarrier we also override the version of client-go to make it properly.